scene/2d/marker_2d.cpp

```diff
@@ -55,19 +55,15 @@ void Marker2D::_draw_cross() {
 	// Axis colors are taken from `axis_x_color` and `axis_y_color` (defined in `editor/editor_themes.cpp`).
 	const Color color_x = Color(0.96, 0.20, 0.32);
 	PackedColorArray colors_x = {
-		color_x,
 		color_x,
 		color_x.lerp(Color(0, 0, 0), 0.5),
-		color_x.lerp(Color(0, 0, 0), 0.5)
 	};
 	draw_multiline_colors(points_x, colors_x);
 
 	const Color color_y = Color(0.53, 0.84, 0.01);
 	PackedColorArray colors_y = {
-		color_y,
 		color_y,
 		color_y.lerp(Color(0, 0, 0), 0.5),
-		color_y.lerp(Color(0, 0, 0), 0.5)
 	};
 	draw_multiline_colors(points_y, colors_y);
```

Comment:
On the topic of fixing up this code, `lerp(Color(0, 0, 0), 0.5)` can be `darkened(0.5)`. I also don't think we need two sets of points, two sets of colors, and two multiline draw calls.